parquet-tools: add page (#4588)
tldr-pages/tldr: pages/common/parquet-tools.md

# parquet-tools

> A tool to show, inspect and manipulate Parquet file.
> More information: <https://github.com/apache/parquet-mr/tree/master/parquet-tools>.

> - Print the content of a Parquet file:

Reviewer: ```suggestion
- Display the content of a Parquet file:
```

> `parquet-tools cat {{path/to/parquet}}`

Reviewer: I installed the `parquet-tools` package in Arch Linux and the commands are in the top level of Bash:
```suggestion
`parquet-cat {{path/to/parquet}}`
```
the same for: `parquet-dump`, `parquet-head`, `parquet-merge`, `parquet-meta`, `parquet-rowcount` and `parquet-schema`.

Author: @navarroaxel 
Thank you for your suggestions.

I installed parquet-tools package in MacOS X using homebrew.
I can't find these commands like parquet-cat.

These commands are only wrapper command for jar file. 
You can call java -jar ... instead of parquet-tools (https://github.com/apache/parquet-mr/tree/master/parquet-tools#run-locally).
It means that we can use another wrapper like `parquet-cat`  or `parquet-tools cat`. 
And parquet-tools is the most major wrapper tools since I can't find parquet-cat in stackoverflow (see, https://stackoverflow.com/search?q=parquet-cat).

So I think my original code is better.

> - Print the first n record of the Parquet file:

Reviewer: ```suggestion
- Display the first few lines of a Parquet file:
```

`parquet-tools head {{path/to/parquet}}`

> - Print the schema of Parquet file(s):

Reviewer: Is this `a list of space-separated Parquet files`?
```suggestion
- Print the schema of a list of space-separated Parquet files:
```

This applies to the other `Parquet file(s)`.

Author: @navarroaxel Thank you for your question.

I've used the expression `file(s)` from the command's usage print.

I checked the result and understand that the expression should be `file` instead of `file(s)`
```
> parquet-tools schema ./test1.parquet ./test2.parquet
Invalid arguments: unknown extra argument "./test2.parquet"

parquet-schema:
Prints the schema of Parquet file(s)
usage: parquet-schema [option...] <input>
where option is one of:
    -d,--detailed      Show detailed information about the schema.
       --debug         Enable debug output
    -h,--help          Show this help string
       --no-color      Disable color output even if supported
    -o,--originalType  Print logical types in OriginalType representation.
where <input> is the parquet file containing the schema to show
```

I'll re-check all command using 'file(s)' expression, and push it.

Author: I re-checked all command whether we put multiple files to the command or not. 
I updated the document. Please check it.

`parquet-tools schema {{path/to/parquet}}`

- Print the metadata of Parquet file(s):

`parquet-tools meta {{path/to/parquet}}`

- Print the content and metadata of a Parquet file:

`parquet-tools dump {{path/to/parquet}}`

> - Merge multiple Parquet files into one:

Reviewer: ```suggestion
- Concatenate several Parquet files into the target one:
```

> `parquet-tools merge {{path/to/input_parquet_1}} {{path/to/input_parquet_2}} {{path/to/output_parquet}}`

Reviewer: Based on the `cat` page:
```suggestion
`parquet-tools merge {{path/to/parquet1}} {{path/to/parquet2}} {{path/to/target_parquet}}`
```
